### PR TITLE
WT-10335 file names overlap

### PR DESCRIPTION
**Pull request by mariammojid**
Renamed C files with overlapping names. Global searched all directories for instances of each file and made the appropriate updates.

### CODE REVIEW

File: test/format/CMakeLists.txt [hunk 1]
```diff
@@ -6,20 +6,20 @@ set(format_sources
     bulk.c
     checkpoint.c
     compact.c
-    config.c
+    form_config.c
```
Comment (korteland) [inline test/format/CMakeLists.txt:9]: Nitpick: can you make the `form_` ones `format_` instead please?

File: test/utility/CMakeLists.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
 project(libtest C)
 
 set(sources
-    misc.c
-    modify.c
+    utility_misc.c
```
Comment (korteland) [inline test/utility/CMakeLists.txt:4]: I think this one is inconsistent, it should be `util_misc.c`.

**Review by korteland — COMMENTED** [review]
This looks pretty good, can you please list which files conflicted with the ones you changed? e.g. for `test/fops/file.c`, which was the other `file.c` that existed? 

(I realise this might be annoying - I'm trying to figure out which file was better to rename and whether you've picked a reasonable one. If it's a pain, let me know and we can figure out something else.)

File: test/format/CMakeLists.txt (hunk 1 above)
Comment (mariammojid) [reply test/format/CMakeLists.txt]: Conflicting files listed with occurrence and updated paths below:

- `config.c `- 3 instances
```
src/config/config.c
bench/wtperf/wtperf_config.c
test/format/format_config.c
```

- `file.c` - 2 instances
```
test/thread/file.c
test/fops/fops_file.c
```

- `config_def` - 2 instances
```
src/config/config_def.c
test/format/format_config_def.c
```

- `salvage.c `- 2 instances
```
test/salvage/salvage.c
test/format/format_salvage.c
```

- `timestamp.c `- 2 instances
```
src/support/timestamp.c
test/format/format_timestamp.c
```

- `util.c `- 2 instances
```
test/csuite/random_directio/util.c
test/format/format_util.c
```

- `misc.c` - 2 instances
```
bench/wtperf/misc.c
test/utility/util_miscs.c
```

- `modify.c `- 2 instances
```
src/support/modify.c
test/utility/util_modify.c
```

- `random.c` - 2 instances
```
test/format/random.c
test/utility/util_random.c
```

File: test/utility/util_miscs.c [hunk 3]
```diff
@@ -552,4 +552,4 @@ example_setup(int argc, char *const *argv)
         home = "WT_HOME";
     testutil_make_work_dir(home);
     return (home);
-}
+}
```
Comment (korteland) [inline test/utility/util_miscs.c:555]: Just a heads-up: when you see this symbol, it means the file is missing a newline at the end. If you're using `git add -p` it probably whinged, but it's easy to miss. You can fix it by adding a newline at the end of the file, or let me know if that's unclear or you need a hand.

File: test/format/CMakeLists.txt (hunk 1 above)
Comment (korteland) [reply test/format/CMakeLists.txt]: Thanks, this is super useful! Given that we already have a `util_misc.c`, would it be better to rename `wtperf/misc.c` to `wtperf/wtperf_misc.c` and leave `test/utility/misc.c` alone?

**Review by korteland — COMMENTED** [review]
This is getting super close, thanks for being so persistent on this :-)

File: test/utility/util_miscs.c (hunk 3 above)
Comment (mariammojid) [reply test/utility/util_miscs.c:555]: Ok cool I'll make those changes

**Review by korteland — APPROVED** [review]
LGTM - thanks!

**Pull request merged**